guides/v2.1/javascript-dev-guide/widgets/widget_modal.md

```diff
@@ -157,7 +159,7 @@ Sets a margin between slide modal windows.
 **Default value**: 45
 
 ### `responsive` {#modal_responsive}
-Turn popup modal window to slide panel on small screens. Available if the `type` option is set to `modal`.
+Turn popup modal window to slide panel on small screens. Available if the `type` option is set to `popup` or `slide` or `custom`.
```

Comment:
Please put a blank line between the heading and the text.

Reply:
@dobooth Do I need to put a blank line after all headings?

Reply:
That would be great.